text-ui-test/EXPECTED.TXT

```diff
@@ -1,3 +1,11 @@
+Hey, I didn't find directory fooddata/
+adding fooddata/ into repository...
+Hey, I didn't find list.txt in fooddata/!
+creating new file...
+Hey, I didn't find directory tasksdata/
+adding tasksdata/ into repository...
+Hey, I didn't find scheduleTasks.txt in tasksdata/!
+creating new file...
```

Comment:
@swatimahadevan the food data folder would be instantiated too, as i modified the `load` functionality